openshift/templates/issuer-admin/issuer-admin-deploy.overrides.sh

```diff
@@ -24,9 +25,9 @@ generateConfigMap "${CADDY_CONFIG_MAP_NAME}${SUFFIX}" "${CADDY_SOURCE_FILE}" "${
 # - DEPLOYMENT_ENV_NAME
 
 # Combine the profile's default config files with its environment specific config files before generating the config map ...
-profileRoot=$( dirname "$0" )/config/${PROFILE}
+profileRoot=$( dirname "$0" )/config
```

Comment:
Was removing support for profile specific configurations intentional?

Reply:
yes, since there is only one profile (BC), I decided to make it the default profile